### DISCUSSION TITLE
Strelka missing

### DISCUSSION THREAD

**Discussion by nc171** [Security-Onion-Solutions/securityonion#11152] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I did a fresh install with SO ISO but I can't seem to get it working. I have a working 2.3 instance but am testing out 2.4 and Strelka is not loaded for some reason. I will paste my screenshots below. It's acting like it didn't get installed. I tried to reload the server twice and once from within the installation of SO. When I run So-Status now it says, "Installation does not appear to be complete. A highstate has not fully completed."

![SO error](https://github.com/Security-Onion-Solutions/securityonion/assets/120204559/20c46467-5775-449c-91a4-a4a042d03def)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120204559/9dac3019-07fe-4737-bbc4-7dd951b803ed)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Did you verify the ISO as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md?

**Comment by nc171** [comment]
To update this, I started from scratch, downloaded and verified the image. I then proceeded with the install using that verified image. The installation process hung and then eventually gave me an error stating the install had a problem. I have attached that error message to the ticket. After that boot, I was able to pull up the webpage but it's still missing So-elestalert, so-elastic-fleet and sl0elasticsearch. I have added a screenshot of that as well. 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120204559/84c332cd-c041-4b20-8044-df044bb4ca0f)

![20230830_062954](https://github.com/Security-Onion-Solutions/securityonion/assets/120204559/dbd9c0ce-8b0c-4b3a-9fd5-4340356088d2)

**Reply by TOoSmOotH** [reply]
How much RAM do you have? What is in the errors log in /root?

**Reply by nc171** [reply]
12gb of RAM in it and I am working on getting those logs to be able to upload them.

**Reply by TOoSmOotH** [reply, marked as answer]
https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments

You need more RAM if you want to run standalone.

**Reply by nc171** [reply]
While I see that further down in the system requirements. My question would be, why would it need 24GB just to do the install? My install failed but I can't imagine it was the result of not enough RAM. I can see it needing it after all the services fired up and were loaded but not during install, or maybe I am missing something about the install process.

**Reply by TOoSmOotH** [reply]
The install process does fire up all the things. It has to load elastic search and elastic fleet to import all the pipelines/integrations etc. If you don't have enough memory elastic search usually gets OOM killed causing a cascade of failures. There is a lot going on during the install process.

**Reply by nc171** [reply]
Thanks for that info. I was able to add RAM to the server to bring it up to 24GB. I reinstalled and it's up and running! Thanks for your help!
